optimum/exporters/tasks.py

```diff
@@ -949,6 +949,10 @@ class TasksManager:
             "text-generation-with-past",
             onnx="GraniteOnnxConfig",
         ),
+        "paligemma": supported_tasks_mapping(
+            "feature-extraction",
```

Comment:
looks like the task here should be `"image-text-to-text"` https://huggingface.co/google/paligemma2-3b-pt-224/blob/main/README.md?code=true#L4


In which case should also be added [here]( https://github.com/huggingface/optimum/blob/4a7cb298140ee9bed968d98a780a950d15bb2935/optimum/exporters/tasks.py#L203)
```
"image-text-to-text": "AutoModelForImageTextToText"
```
 
and here 

https://github.com/huggingface/optimum/blob/4a7cb298140ee9bed968d98a780a950d15bb2935/optimum/exporters/onnx/base.py#L144